plugins/modules/ec2.py

```diff
@@ -577,6 +577,297 @@
 
 '''
 
+RETURN = r'''
+changed:
+    description: If the EC2 instance has changed.
+    type: bool
+    returned: always
+    sample: true
+instances:
+    description: The instances.
+    type: list
+    returned: always
+    contains:
+        ami_launch_index:
+            description: The AMI launch index, which can be used to find this instance in the launch group.
+            type: int
+            returned: always
+            sample: 0
+        architecture:
+            description: The architecture of the image.
+            type: str
+            returned: always
+            sample: "x86_64"
+        block_device_mapping:
+            description: Any block device mapping entries for the instance.
+            type: dict
+            returned: always
+            sample: {
+                "/dev/xvda": {
+                    "delete_on_termination": true,
+                    "status": "attached",
+                    "volume_id": "vol-06d364586f5550b62"
+                }
+            }
+        dns_name:
+            description: If the EC2 instance has changed.
```

Comment:
```suggestion
            description: The public DNS name assigned to the instance.
```
I'm reasonably certain it's the public address at least, http://boto.cloudhackers.com/en/latest/ref/ec2.html#boto.ec2.instance.Instance doesn't document dns_name.

Reply:
Yes, you're right. In the copy/paste process I forget to rectify. Thank you.